### DISCUSSION TITLE
Multi-Tenant Domains

### DISCUSSION THREAD

**Discussion by activenode** [supabase/supabase#13693] [Feature Requests]
As of https://github.com/supabase/supabase/issues/12429#issue-1585154640 there is still the request to enable multiple custom domains not just one.

So basically this Feature Request is to think about multi-tenancy in accordance to the other multi-tenancy issues that exist.

Even for me this sounds like a bad description for a feature request but it feels like duplication at the same time it is not.

tldr: Enable to be able to add multiple custom domains instead of just one (come up with a clever architecture to do so).
